src/StepInput/StepInput.js

```diff
@@ -25,11 +36,19 @@ const StepInput = React.forwardRef(({
 }, ref) => {
     const [inputValue, updateInputValue] = useState(value);
 
-    const inputGroupClasses = classnames(
+    useEffect(() => {
+        if (!disableStyles) {
+            require('fundamental-styles/dist/step-input.css');
+        }
+    }, []);
+
+    const stepInputClasses = classnames(
         className,
-        'fd-input-group--control',
+        'fd-step-input',
         {
+            'fd-step-input--compact': compact,
             'is-disabled': disabled,
+            'is-readonly': readOnly,
             [`is-${validationState?.state}`]: validationState ?.state
```

Comment:
I wonder why eslint doesn't pick up on this line having an extra space `validationState ?.state`

Reply:
I'm not sure - good catch though!